frontend/playwright.d2d.config.ts

```diff
@@ -4,18 +4,16 @@ import commonConfig from "./playwright.common.config";
 
 function returnWebserverCommand(): string {
   if (process.env.CI) {
-    // CI: use frontend build and backend build, reset and seed database
-    return `cd ../builds/backend &&
-      ./api --reset-database --seed-data --frontend-dist ../frontend --port 8081`;
+    // CI: use existing backend build, reset and seed database
+    return `../builds/backend/api --reset-database --seed-data --port 8081`;
   } else if (process.env.LOCAL_CI) {
-    // LOCAL CI: build frontend, build backend, reset and seed playwright-specific database
+    // LOCAL CI: build frontend, then build and run backend with database reset and seed playwright-specific database
     return `npm run build &&
       cd ../backend &&
-      cargo build &&
-      target/debug/api --database target/debug/playwright.sqlite --reset-database --seed-data --frontend-dist ../frontend/dist --port 8081`;
+    ASSET_DIR=$PWD/../frontend/dist cargo run --features memory-serve -- --database target/debug/playwright.sqlite --reset-database --seed-data --port 8081`;
```

Comment:
In our `package.json` we use `cross-env` for environment variables. So that makes me wonder if it's needed here as well.

Reply:
I'm actually not sure. Slapping `cross-env` in front of it doesn't seem to break the config, but I'm not sure if it's actually doing anything. I guess since this is indirectly run via `npm`, it knows about `cross-env`?
@praseodym @oliver3 @rnijveld @marlonbaeten can you verify this?

Reply:
[e472080](https://github.com/kiesraad/abacus/pull/429/commits/e472080abcb202e2ecc3b5d63965488cfdc27186)